optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -886,7 +886,7 @@ def __call__(
             timestep_cond = None
             if self.unet.config.time_cond_proj_dim is not None:
                 guidance_scale_tensor = torch.tensor(self.guidance_scale - 1).repeat(
-                    batch_size * num_images_per_prompt
+                    num_prompts * num_images_per_prompt
```

Comment:
I personally think we should submit this as a separate PR.  I also noted some small fixes we could create for the original SD.  So we can submit this separately as its not part of SDXL.  However, lets not forget, and lets merge with few other smaller fixes (I think there are some in the callbacks) as well as some typos

Reply:
Agreed. I'll revert the change and we can submit in separate PR.